frontend/public/index.html

```diff
@@ -2,11 +2,6 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```

Comment:
BTW can we keep the default favicon and replace it later using Helmet? I mean than this line would be here and it will act as a default fallback?

Reply:
All of this has been returned to `index.html`

`  <meta charset="utf-8" />
    <link rel="icon" href="%PUBLIC_URL%/favicon" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="theme-color" content="#000000" />
    <meta name="description" content="Casper Labs CSPR Block Explorer" />
    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />`